documentation/docs/steps/checkChangeInDevelopment.md

```diff
@@ -6,6 +6,37 @@
 
 * No prerequisites
```

Comment:
Is it possible to leave the prerequisites section completely out, since there are no prerequisites? Also, the results of
 ${docGenStepName} and ${docGenDescription} don't appear in the build pages